ui/handle_input.go

```diff
@@ -418,3 +452,31 @@ func (win *Window) sortTransactions() {
 		return frontTime.Before(backTime)
 	})
 }
+
+func (win *Window) updateToTransactionDetailsBtns() {
+	win.inputs.toTransactionDetails = make(map[string]*gesture.Click)
```

Comment:
since you know the size of the transactions, you can declare the size of the map.